keystrokes/src/components/Auth/login.js

```diff
@@ -0,0 +1,29 @@
+import React, { Component } from 'react';
+import { FirebaseAuth } from 'react-firebaseui';
+import { auth, googleProvider, emailProvider } from '../../config/constants';
+
+class Login extends Component {
+	constructor(props) {
+		super(props);
+
+		this.uiConfig = {
```

Comment:
Putting the config on this is fine, but unnecessary. you could define it as a const outside of the component alternatively